openface/install.bash

```diff
@@ -1,5 +1,11 @@
 #! /usr/bin/env bash
 
+if [ "$CI" == "true" ]
```

Comment:
Why?

Reply:
People recognition starts installing this and gets stuck at stdin command towards the end and times out

Reply:
Either we clean this target up or don't run it in CI